odoo/models.py

```diff
@@ -511,7 +509,6 @@ class BaseModel(metaclass=MetaModel):
     _date_name = 'date'         #: field to use for default calendar view
     _fold_name = 'fold'         #: field to determine folded groups in kanban views
 
-    _needaction = False         # whether the model supports "need actions" (Old API)
```

Comment:
Adding a warning that that is a dead attribute to the metaclass could be nice as well. Also for the now-dead view_init.